frontend/src/components/SearchBar.jsx

```diff
@@ -1,60 +1,64 @@
 import { Box, IconButton, TextField } from "@mui/material";
 import { Autocomplete } from "@react-google-maps/api";
-import { Menu, Settings } from "@mui/icons-material";
+import { Settings } from "@mui/icons-material";
 import { useState } from "react";
-import PropTypes from 'prop-types';
+import PropTypes from "prop-types";
 import { useOutletContext } from "react-router-dom";
 
-function SearchBar({ onPlaceSelected, toggleDrawer }) {
-    const [autocomplete, setAutocomplete] = useState(null);
-    const { handleOpenSettings } = useOutletContext();
+function SearchBar({ onPlaceSelected }) {
+  const [autocomplete, setAutocomplete] = useState(null);
+  const { handleOpenSettings } = useOutletContext();
 
-    const onLoad = (instance) => {
-        setAutocomplete(instance);
-    };
+  const onLoad = (instance) => {
+    setAutocomplete(instance);
+  };
 
-    const onPlaceChanged = () => {
-        if (autocomplete !== null) {
-            const place = autocomplete.getPlace();
-            onPlaceSelected
-            onPlaceSelected(place);
-        } else {
-            console.error("Autocomplete is not loaded yet!");
-        }
-    };
+  const onPlaceChanged = () => {
+    if (autocomplete !== null) {
+      const place = autocomplete.getPlace();
+      onPlaceSelected;
+      onPlaceSelected(place);
+    } else {
+      console.error("Autocomplete is not loaded yet!");
+    }
+  };
 
-    return (
-        <Box display="flex" alignItems="center" mt={2} p={1}
-            width={{ xs: '100%', sm: 'auto', md: '400px'}}
-            sx={{ bgcolor: 'var(--color-light)', borderColor: 'divider', 
-                borderRadius: 3, boxShadow: 1, gap: 1, zIndex: 10 }}
-            >
-            <IconButton onClick={toggleDrawer}>
```

Comment:
The drawer/sidebar on the left can be used to store user search queries, results, settings, etc.. Although those features could be moved to right pane, it's better to just leave it here for now and we could discuss and decide a place later.